plugins/rst.lisp

```diff
@@ -4,20 +4,37 @@
 (defpackage :coleslaw-rst
   (:use :cl)
   (:import-from :coleslaw #:render-text)
-  (:import-from :docutils #:read-rst #:write-part #:register-settings-spec
-                          #:visit-node #:write-document)
-  (:import-from :docutils.writer.html #:html-writer #:write-part)
+  (:import-from :docutils
+                #:document
+                #:read-rst
+                #:register-settings-spec
+                #:visit-node
+                #:write-document)
+  (:import-from #:docutils.writer.html
+                #:html-writer
+                #:parts
+                #:body-pre-docinfo
+                #:body
+                 )
   (:export #:enable))
 
 (in-package :coleslaw-rst)
 
+
+(defmethod visit-node :after ((writer html-writer) (document document))
+  "This method removes unnecessary HTML elements, such as html, head, body
+and make docutils output only html fragment with document itself."
+  
+  (setf (slot-value writer 'parts) '(body-pre-docinfo
```

Comment:
Could still just be changed to mention all of the parts, but not set any values for `head-prefix`, `body-prefix` and `body-suffix`. It's a bit more declarative than adding a new method, but then again, I don't think that matters too much.